Server/Game/tests.py

```diff
@@ -155,6 +156,55 @@ def test3_set_team_valid_input(self):
 
 		endTestLog("test3_set_team_valid_input")
 
+	def test4_find_match_before_set_team(self):
+		startTestLog("test4_find_match_before_set_team")
+		# Create user and login
+		username = "set_team_user"
+		self.assertTrue(self.channel.createUserAndLogin(
+			{"username":username,"password":"abc12345","email":"setTeam@email.com"}))
+
+		# Call find match
+		self.channel.send('{"Command":"FM"}')
+		result = json.loads(self.channel.receive())
+		self.assertEqual(result["Success"], False)
+
```

Comment:
Can you also add an assert for the error message sent back for this failure case?  So in this case, assert that we receive "You must set a team before starting a match." back.